src/frontend/Sider/Behandling/Inngangsvilkår/Aktivitet/Aktivitet.tsx

```diff
@@ -38,6 +38,10 @@ const Aktivitet: React.FC = () => {
 
     const skalViseTabell = aktiviteter.length > 0 || leggerTilNyPeriode;
 
+    const erIStegInngangsvilkår = behandling.steg === 'INNGANGSVILKÅR';
```

Comment:
For å ikke sjekke på mange sider om `behandling.steg === 'INNGANGSVILKÅR'`, burde vi ha en "StegContext" som tar inn behandling og steg for gitt side? 
Den kan då eksponere "stegErRedigerbart" eller noe slikt

Reply:
➕

Reply:
Hvordan skal det funke? Da må man vel enten ha en context for hver av fanene, eller utlede hvilken fane man står i fra pathen?

Reply:
Trenger det egentlig å være en egen context? Eller kunne bare behandling returnert informasjonen om steget? Det ligger jo på en behandling og må være på samme nivå. Er egentlig relativt likt som `erBehandlingRedigerbar`

Kunne også vurdert å ha en "samlet versjon" av disse to. Fordi veldig mange steder kommer vi til å måtte sjekke `erBehandlingRedigerbar && riktigSteg`, men det er litt overkill fordi `riktigSteg` er jo bare tidligere steg enn om den er redigerbar

Reply:
For å vite hvilket steg man er i i frontend (hvilken fane saksbehandler står i) må vi utlede det fra fanenmenyen. Synes det blir litt rotete å blande fanene inn i behandlingscontext så lager heller egen context for dette.

Reply:
https://github.com/navikt/tilleggsstonader-sak-frontend/pull/234/commits/0ef605a3b1efc9892a01fedefb779ebd8a509afc